mir-module/src/test/java/org/mycore/mir/acl/MIRACLTest.java

```diff
@@ -47,7 +46,7 @@ public void disableDebug() {
     @Override
     public void setUp() throws Exception {
         super.setUp();
-        accessSystem = MCRConfiguration2.<MCRFactsAccessSystem>getInstanceOf("MCR.Access.Class")
+        accessSystem = MCRConfiguration2.getInstanceOf(MCRFactsAccessSystem.class, "MCR.Access.Class")
```

Comment:
should rather use `getInstanceOfOrThrow` here, in order to obtain a more informative error message